.gitignore

```diff
@@ -1,10 +1,9 @@
-*~
-*.pyc
-
-.sass-cache/
+.DS_STORE
 _site/
-.bundle/
-
-# Ignore locked gems because GitHub Pages controls the gems in production, not Nuclide. Omit the
-# lock so `bundle install` always installs the latest.
-Gemfile.lock
```

Comment:
Yes. In general, it is good to commit a `Gemfile.lock` so that it is clear the current version of dependencies. If you are working on a Gem or something, then you wouldn't normally do this. We commit `Gemfile.lock` in most of our docs.